src/token/erc721/erc721.cairo

```diff
@@ -133,26 +154,32 @@ mod ERC721 {
 
     #[external(v0)]
     impl ERC721Impl of interface::IERC721<ContractState> {
+        /// Returns the number of NFTs owned by `account`.
         fn balance_of(self: @ContractState, account: ContractAddress) -> u256 {
             assert(!account.is_zero(), Errors::INVALID_ACCOUNT);
             self.ERC721_balances.read(account)
         }
 
+        /// Returns the owner address of `token_id`.
         fn owner_of(self: @ContractState, token_id: u256) -> ContractAddress {
             self._owner_of(token_id)
         }
 
+        /// Returns the address approved for `token_id`.
         fn get_approved(self: @ContractState, token_id: u256) -> ContractAddress {
             assert(self._exists(token_id), Errors::INVALID_TOKEN_ID);
             self.ERC721_token_approvals.read(token_id)
         }
 
+        /// Query if `operator` is an authorized operator for `owner`.
         fn is_approved_for_all(
             self: @ContractState, owner: ContractAddress, operator: ContractAddress
         ) -> bool {
             self.ERC721_operator_approvals.read((owner, operator))
         }
 
+        /// Change or reaffirm the approved address for an NFT.
+        /// Emits an [Approval](Approval) event.
```

Comment:
```suggestion
        /// Emits an `Approval` event.
```

This syntax of `[]()` is for solidity doc gen tools, and we shouldn't replicate it (or we should document it).

Applies for the rest of the file as well.